charts/cert-manager/values.yaml

```diff
@@ -0,0 +1,21 @@
+# Values passed to official cert-manager chart
+cert-manager:
+  global:
+    podSecurityPolicy:
+      enabled: true
+    
+  extraArgs:
+  - --enable-certificate-owner-ref=true
+
+# Email that will be used to register issuer with lets encrypt
+# Will receive communications from lets encrypt here
+issuerAccountEmail: ap-devops@broadinstitute.org
+dnsProject:
+  # Id for the GCP project that will host your cloud DNS resources
+  id: dsp-devops
+  serviceAccount:
+    # name of the k8s secret that will hold projects SA credentials 
+    secretName: something
```

Comment:
A bunch of these values really shouldn't have defaults, but should always be required to be overwritten in values files. The pattern I've been using for such values is:
- Comment out the value in the chart
- Add a [required block](https://helm.sh/docs/howto/charts_tips_and_tricks/#know-your-template-functions) around anywhere it is used so that there's a helpful error message if it's omitted.

Reply:
ty for the suggestions! will make these changes.